_posts/2022-09-09-release.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Release 2022.35
+author: Priya
+date: 2022-09-09
+category: Jekyll
+layout: post
+---
+### Enhancements
+
+#### New column browser integrated with action bar
+Integrated new column browser with dataset action bar that combines the functionalities of previous "Column browser" and "Explore columns". We will soon remove the "Explore columns" feature.
+
+#### Storage based restrictions
+Mammoth introduced storage-based restrictions to limit certain activities when an account reaches its storage limit. This is controlled via split, and if disabled, storage quota is not considered. For more details, refer to [Restrictions](https://mammoth.atlassian.net/wiki/spaces/PD/pages/2245099615/Storage+based+Restrictions).
```

Comment:
For more details should be taken out. The link is internal to Mammoth

Reply:
Fixed